src/components/chart/EatChart.tsx

```diff
@@ -0,0 +1,123 @@
+import { useEffect, useState } from 'react';
+import { formatDate, getDateString } from '@/functions/common';
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Title,
+  Tooltip,
+  Legend
+} from 'chart.js';
+import { Bar } from 'react-chartjs-2';
+import { getFoodOfUser } from '@/functions/eatCycle';
+import ChartDataLabels from 'chartjs-plugin-datalabels';
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Title,
+  Tooltip,
+  Legend,
+  ChartDataLabels
+);
+
+export default function EatChart(props: {
+  onLoad: (loading: boolean) => void,
+  startDate?: Date,
```

Comment:
function requires currentDate so change the variable name from startDate to currentDate

Reply:
fixed kub